scm/doc/TechGuide/chap_quick.rst

```diff
@@ -858,13 +870,17 @@ Quickstart application installed with Docker Toolbox.
       docker run --rm -it -v ${OUT_DIR}:/home --name run-ccpp-scm ccpp-scm ./run_scm.py -c twpice -d
 
    will run through the TWPICE case using the default suite and namelist
-   and put the output in the shared directory. NOTE: Windows users may
-   need to omit the curly braces around environment variables: use ``$OUT_DIR``
-   instead of ``${OUT_DIR}``. For running through all supported cases and suites, use
+   and put the output in the shared directory.
+
+   .. note::
+     Windows users may need to omit the curly braces around environment variables: use ``$OUT_DIR``
+     instead of ``${OUT_DIR}``. 
+
+   For running through all supported cases and suites, use
```

Comment:
The rt_test_cases.py has more than just the permutations of supported cases/suites, but that is probably not a big deal.